canonical/maas-ui
# remove unused machine.list properties

## Done

- remove unused machine.list properties and update types per spec https://docs.google.com/document/d/1cDQxuEdNHkrk95Y9Qn0iiUMUerE90itEQ_eTzcGfK-k/edit#heading=h.rje15jrsfn6d

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to machine list page
- Select a few machines
- Verify that you can open the Release form
- Go to machine details page
- Verify that you can open the Release form
- Go to KVM page and verify that a list of machines is displayed correctly


## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against _main_ rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in main.

## Screenshots

It could be helpful to provide some screenshots to aid in QAing the change.
